api/ai/new/interfaces/team_generation_options.py

```diff
@@ -27,7 +27,9 @@ def validate(self):
             )
         if self.initial_teams:
             Schema([TeamShell]).validate(self.initial_teams)
-            if len(self.initial_teams) != self.total_teams:
+            if not is_unique([t.id for t in self.initial_teams]):
+                raise ValueError(f"Ids of team shells must be unique!")
+            if len(self.initial_teams) > self.total_teams:
                 raise ValueError(
-                    f"team_options size ({len(self.initial_teams)}) != total_teams ({self.total_teams})"
+                    f"team_options size ({len(self.initial_teams)}) > total_teams ({self.total_teams})"
```

Comment:
Question: So we allow passing initial teams with length less then total teams?

Reply:
Yep! Reasoning given in PR description